e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -138,36 +138,55 @@ Scenario('local authority enters children @create-case-with-mandatory-sections-o
   I.seeAnswerInTab(21, 'Party', 'Give details, including assessment outcomes and referrals to health services', 'mock reason');
 
-  I.seeAnswerInTab(3, 'Party', 'First name', 'Susan');
-  I.seeAnswerInTab(4, 'Party', 'Last name', 'Wilson');
-  I.seeAnswerInTab(5, 'Party', 'Date of birth', '1 Jul 2016');
-  I.seeAnswerInTab(6, 'Party', 'Gender', 'Girl');
-  I.seeAnswerInTab(7, 'Party', 'Child\'s living situation', 'Living with respondents');
-  I.seeAnswerInTab(8, 'Party', 'What date did they start staying here?', '2 Nov 2017');
+  I.seeAnswerInTab(1, 'Party', 'First name', 'Susan');
+  I.seeAnswerInTab(2, 'Party', 'Last name', 'Wilson');
+  I.seeAnswerInTab(3, 'Party', 'Date of birth', '1 Jul 2016');
+  I.seeAnswerInTab(4, 'Party', 'Gender', 'Girl');
+  I.seeAnswerInTab(5, 'Party', 'Child\'s living situation', 'Living with respondents');
+  I.seeAnswerInTab(6, 'Party', 'What date did they start staying here?', '2 Nov 2017');
+  I.seeAnswerInTab(7, 'Party', 'Key dates for this child', 'Tuesday the 11th');
+  I.seeAnswerInTab(8, 'Party', 'Brief summary of care and contact plan', 'care plan summary');
+  I.seeAnswerInTab(9, 'Party', 'Are you considering adoption at this stage?', 'No');
+  I.seeAnswerInTab(10, 'Party', 'Mother\'s full name', 'Laura Smith');
+  I.seeAnswerInTab(11, 'Party', 'Father\'s full name', 'David Smith');
+  I.seeAnswerInTab(12, 'Party', 'Does the father have parental responsibility?', 'Yes');
+  I.seeAnswerInTab(13, 'Party', 'Name of social worker', 'James Jackson');
+  I.seeAnswerInTab(1, 'Social worker\'s telephone number', 'Telephone number', '01234567');
+  I.seeAnswerInTab(15, 'Party', 'Does the child have any additional needs?', 'No');
+  I.seeAnswerInTab(16, 'Party', 'Do you need contact details hidden from other parties?', 'Yes');
+  I.seeAnswerInTab(17, 'Party', 'Do you believe this child will have problems with litigation capacity (understanding what\'s happening in the case)', 'No');
+
+  caseViewPage.selectTab(caseViewPage.tabs.confidential);
+  I.seeAnswerInTab(1, 'Party', 'First name', 'Susan');
+  I.seeAnswerInTab(2, 'Party', 'Last name', 'Wilson');
+  I.seeAnswerInTab(3, 'Party', 'Date of birth', '1 Jul 2016');
+  I.seeAnswerInTab(4, 'Party', 'Gender', 'Girl');
+  I.seeAnswerInTab(5, 'Party', 'Child\'s living situation', 'Living with respondents');
+  I.seeAnswerInTab(6, 'Party', 'What date did they start staying here?', '2 Nov 2017');
   I.seeAnswerInTab(1, 'Current address', 'Building and Street', '2 Three Tuns Wynd');
   I.seeAnswerInTab(2, 'Current address', '', 'High Street');
   I.seeAnswerInTab(3, 'Current address', '', 'Stokesley');
   I.seeAnswerInTab(4, 'Current address', 'Town or City', 'Middlesbrough');
   I.seeAnswerInTab(5, 'Current address', 'Postcode/Zipcode', 'TS9 5DQ');
   I.seeAnswerInTab(6, 'Current address', 'Country', 'United Kingdom');
```

Comment:
this will add some complexity to the test. I would need to login as hmcts admin and manage representatives to get someone who would not be able to see this

Reply:
Sure, we might ask Raghu/Samantha to help decide how this could be tested 👹